central/processindicator/store/store.go

```diff
@@ -21,6 +21,7 @@ type Store interface {
 
 	UpsertMany(context.Context, []*storage.ProcessIndicator) error
 	DeleteMany(ctx context.Context, id []string) error
+	PruneMany(ctx context.Context, id []string) error
```

Comment:
I suppose the goal of exposing this function here was to use it in the `pruneIndicators` function of the associated datastore layer. This seems to be missing from the current state of the PR.

Reply:
That was the goal and then changed my mind when I saw how that pruning was implemented in the datastore.  So this is a left over artifact from that choice.